dashboard/app/controllers/javabuilder_connection_test_controller.rb

```diff
@@ -0,0 +1,23 @@
+require 'cdo/firehose'
+
+class JavabuilderConnectionTestController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
what happens if we hit this route for an unauthenticated user?

Reply:
I misunderstood this before_action and thought it returned 403 if the user wasn't signed in, instead of redirecting. I've updated the controller to just respond forbidden if anyone tries to hit this route signed out (although this should not happen as implemented currently).